types/block.go

```diff
@@ -83,6 +84,7 @@ type Data struct {
 	Txs                    Txs
 	IntermediateStateRoots IntermediateStateRoots
 	Evidence               EvidenceData
+	ConsensusMessages      []*proto.Any
```

Comment:
@faultytolly and @testinginprod I think we discussed that we need  a way for full nodes to validate correctness of last consensus msgs they recive and we discussed if we should add a specific ConsensusMessagesHash but we agreed on it being part of the header hash, but it doesn't seen like part of the header and it's not being part of the LastDataHash. 

We can leave it as an open issue for now but just making sure this is a concious decision vs a miss.

Reply:
adding to this - we do check consensusMsgs Response as its part of the blockResultsHash. so I guess it's fine for now.